MinecraftClient/Commands/Inventory.cs

```diff
@@ -86,8 +84,21 @@ public override string Run(McClient handler, string command, Dictionary<string,
                                 {
                                     string displayName = item.Value.DisplayName;
                                     if (String.IsNullOrEmpty(displayName))
-                                        response.Add(String.Format(" #{0}: {1} x{2}", item.Key, item.Value.Type, item.Value.Count));
-                                    else response.Add(String.Format(" #{0}: {1} x{2} - {3}§8", item.Key, item.Value.Type, item.Value.Count, displayName));
+                                    {
+                                        Dictionary<string, Object> itemnbt = item.Value.NBT;
+                                        if (itemnbt.ContainsKey("Damage"))
```

Comment:
Instead of implementing NBT parsing in the command, I think it would be better to implement a `Damage` getter in `Item.cs` just like [`DisplayName`](https://github.com/ORelio/Minecraft-Console-Client/blob/4cc29a6ee662613eb3e4d916cc5bda3f699a029a/MinecraftClient/Inventory/Item.cs#L56). The getter could return `0` or `-1` if no NBT damage is set for the item (as you prefer).